scripts/base/files/x509/main.zeek

```diff
@@ -43,7 +43,7 @@ export {
 		## All extensions in the order they were raised.
 		## This is used for caching certificates that are commonly
 		## encountered and should not be relied on in user scripts.
-		extensions_cache: vector of any &default=vector();
+		extensions_cache: table[count] of any;
```

Comment:
Is there a reason that we have to change this from a vector to a table of count?

From a semantic perspective I like a vector quite a bit more here; using a table[count] seems to return this to the old use of count tables (like we used it for string indexing) that we have tried to get away from.

I admittedly have a suspicion - this probably is one of the few places of zeek where different record types are stuck into a single vector. I still like the original syntax more :)

Reply:
Yes, this is due to vector-of-any's needing to be homogeneous.  I agree that the original syntax is somewhat more pleasing, though I also was happy to find that making this switch required only very minor tweaks.